torch_optimizer/__init__.py

```diff
@@ -33,3 +35,57 @@
     'RangerVA'
 )
 __version__ = '0.0.1a11'
+
+
+package_opts = [
+    AccSGD,
+    AdaBound,
+    AdaMod,
+    DiffGrad,
+    Lamb,
+    Lookahead,
+    NovoGrad,
+    PID,
+    QHAdam,
+    QHM,
+    RAdam,
+    SGDW,
+    Yogi,
+    Ranger,
+    RangerQH,
+    RangerVA,
+]
+
+builtin_opts = [
```

Comment:
I thought it makes sense to separate between local optimizers and builtin optimizers. I can also make a large list if needed.
Currently, this is possible 
```opt_class = torch_optimizer.get('adam')```
but this is 
```from torch_optimizer import Adam```.
Would you like to enable the second behavior?

Reply:
I do not think this is good idea, it is a bit misleading and if one wants this, it is super easy to build for local codebase. If one day we want own Adam implementation names will collide.